client/packages/programs/src/JsonForms/common/JsonForm.tsx

```diff
@@ -227,6 +231,23 @@ export const JsonForm: FC<PropsWithChildren<JsonFormProps>> = ({
       width="100%"
       gap={2}
       paddingX={10}
+      sx={{
+        alignItems: 'flex-start',
+        '& .MuiGrid-container': {
+          '& .MuiGrid-grid-xs-true': {
+            '& .MuiGrid-container': {
+              '& .MuiGrid-grid-xs-true': {
+                maxWidth: FORM_COLUMN_MAX_WIDTH,
+              },
+            },
+          },
+        },
```

Comment:
There *must* be a better way to target inner containers, right? *Right?*

Reply:
rather than the pretty nesting you can just target directly

```
'& .MuiGrid-container > .MuiGrid-grid-xs-true .MuiGrid-container > .MuiGrid-grid-xs-true':
          {
            maxWidth: FORM_COLUMN_MAX_WIDTH,
          },
```

I'm concerned about being so specific though, with the use of the `MuiGrid-grid-xs-true` class name.
for a similar effect, but more targeted, why not add a class to the `InputWithLabelRow`:

```
<Box
      className="input-with-label-row"
      display="flex"
      ...
```

and then

```
' & .input-with-label-row': {
          maxWidth: FORM_COLUMN_MAX_WIDTH,
        },
```

Reply:
Great suggestion, thanks. Adding a className to the InputRow seems to be the way to go: 636c707c3